src/main/java/com/exasol/containers/ExasolContainer.java

```diff
@@ -227,7 +238,17 @@ public String getDriverClassName() {
 
     @Override
     public String getJdbcUrl() {
-        return "jdbc:exa:" + getContainerIpAddress() + ":" + getMappedPort(getFirstDatabasePort());
+        return "jdbc:exa:" + getContainerIpAddress() + ":" + getFirstMappedDatabasePort()
+                + ";validateservercertificate=0";
+    }
+
+    /**
+     * Get the port to which the first database port in the Exasol configuration is mapped.
+     *
+     * @return mapped first database port
+     */
+    public Integer getFirstMappedDatabasePort() {
```

Comment:
does this metthod need to be public?

Reply:
It does not need to be, but it is useful, since this is the default database port that a client will use from outside the container in 99% of all test cases.